src/daemon/ubuntu_image_host.cpp

```diff
@@ -42,27 +43,27 @@ namespace
 {
 constexpr auto index_path = "streams/v1/index.json";
 
-auto download_manifest(const QString& host_url, mp::URLDownloader* url_downloader)
+bool is_default_constructed(const std::pair<std::string, std::unique_ptr<mp::SimpleStreamsManifest>>& manifest_pair)
 {
-    auto json_index = url_downloader->download({host_url + index_path});
+    return manifest_pair == std::pair<std::string, std::unique_ptr<mp::SimpleStreamsManifest>>{};
+}
+
+auto download_manifest(const QString& host_url, mp::URLDownloader* url_downloader, bool is_force_update_from_network)
+{
+    auto json_index = url_downloader->download({host_url + index_path}, is_force_update_from_network);
     auto index = mp::SimpleStreamsIndex::fromJson(json_index);
 
-    auto json_manifest = url_downloader->download({host_url + index.manifest_path});
+    auto json_manifest = url_downloader->download({host_url + index.manifest_path}, is_force_update_from_network);
     return json_manifest;
 }
 
 mp::VMImageInfo with_location_fully_resolved(const QString& host_url, const mp::VMImageInfo& info)
 {
-    return {info.aliases,
-            info.os,
-            info.release,
-            info.release_title,
-            info.supported,
-            host_url + info.image_location,
-            info.id,
-            info.stream_location,
-            info.version,
-            info.size,
+    return {info.aliases,   info.os,
+            info.release,   info.release_title,
+            info.supported, host_url + info.image_location,
+            info.id,        info.stream_location,
+            info.version,   info.size,
```

Comment:
Ugh, I assume this is `clang-format` doing this, right?

Reply:
Yep, actually, clang-format is the thing I have been meaning to discuss about. There are some parameters can be fine-tuned to format the file better, this code snippet is one example that can be improved.

Reply:
I really don't see a change, so ideally, as mentioned before, `clang-format` should only run on the diff, not the whole file to avoid these types of things that make it slightly harder to review.